### PR TITLE
Support saving and loading frame tracks as part of presets

### PR DESCRIPTION
**Pull request by ronaldfw**
Frame tracks based on hooked functions are now also stored in and
loaded from presets.

Bug: http://b/173490995

Tested: Manual tests: load/save presets with frame tracks; load/save
presets without frame tracks, both during different states of Orbit
including before and after capture.

### CODE REVIEW

File: OrbitGl/App.cpp [hunk 1]
```diff
@@ -1196,8 +1261,23 @@ void OrbitApp::UpdateProcessAndModuleList(int32_t pid) {
...
+        const ModuleData* module = module_manager_->GetModuleByPath(func.loaded_module_path());
+        if (!process->IsModuleLoaded(module->file_path())) {
+          RemoveFrameTrack(func);
+        } else {
+          if (!module->is_loaded()) {
+            RemoveFrameTrack(func);
+            frame_track_function_hashes_map[module->file_path()].push_back(
+                FunctionUtils::GetHash(func));
+          }
+        }
```
Comment (IrinaShkviro) [inline OrbitGl/App.cpp:1277]:
```suggestion
        } else if (!module->is_loaded()) {
            RemoveFrameTrack(func);
            frame_track_function_hashes_map[module->file_path()].push_back(
                FunctionUtils::GetHash(func));
        }
```
Also maybe it worth to add an explanatory comment for 'if' branches.

**Review by IrinaShkviro — APPROVED** [review]
Looks great from my perspective!

File: OrbitGl/App.cpp (hunk 1 above)
Comment (ronaldfw) [reply OrbitGl/App.cpp]: Thanks, good suggestion. Fixed.

**Review by antonrohr — APPROVED** [review]
code lgtm and I also tested it with one frame track based on vkQueueSubmit.

**Comment by ronaldfw** [comment]
Thanks for the reviews, @IrinaShkviro and @antonrohr!

**Pull request merged**